openfisca_france/model/prestations/minima_sociaux/cmu.py

```diff
@@ -275,7 +276,7 @@ def function(self, simulation, period):
 
         res = cmu_br_i_par[CHEF] + cmu_br_i_par[PART] + forfait_logement
 
-        res += 12 * (aspa + ass + asi + af + cf + asf + paje_clca)
+        res += 12 * (aspa + ass + asi + af + cf + asf + paje_clca + paje_prepare)
```

Comment:
Actually I don't see how to do it using something cleaner. 
@benjello, maybe I can call you to discuss it ?